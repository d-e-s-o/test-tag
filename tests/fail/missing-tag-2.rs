// Copyright (C) 2024 Daniel Mueller <deso@posteo.net>
// SPDX-License-Identifier: (Apache-2.0 OR MIT)

/// Missing tags in second attribute.
#[test_tag::tag(tag)]
#[test_tag::tag]
#[test]
fn cant_compile() {}
