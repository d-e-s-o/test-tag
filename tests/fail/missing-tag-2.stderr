error: expected identifier
 --> tests/fail/missing-tag-2.rs:6:3
  |
6 | #[test_tag::tag]
  |   ^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fail/missing-tag-2.rs:8:21
  |
8 | fn cant_compile() {}
  |                     ^ consider adding a `main` function to `$DIR/tests/fail/missing-tag-2.rs`
